x/incentives/keeper/hooks.go

```diff
@@ -25,12 +26,17 @@ func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochIdentifier string, epochNumb
 			}
 		}
 
-		// if len(gauges) > 10 {
-		// 	ctx.EventManager().IncreaseCapacity(2e6)
-		// }
+		// get active, non-perpetual gauges
+		// perpetual gauges are paid directly by the protocol (i.e streamer)
+		newGauges := make([]types.Gauge, 0, len(gauges))
```

Comment:
super nit: i guess most of the gauges will be perpetual, so maybe it's worth doing `make([]types.Gauge, 0)` to avoid runtime overhead. otherwise runtime will try to find a memory slice for `len(gauges)` though in fact we need much less.